monai/apps/deepgrow/dataset.py

```diff
@@ -0,0 +1,270 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+from typing import Dict, List
+
+import numpy as np
+
+from monai.transforms import AsChannelFirstd, Compose, LoadImaged, Orientationd, Spacingd
+from monai.utils import GridSampleMode
+
+
+def create_dataset(
```

Comment:
I think this is `create_datalist` instead of `create_dataset`, right?

Reply:
it's actually list (json).. from an existing dataset.. dataset possibly talks about Dataset (e.g. Cached, Persistence etc..) and this fuction doesn't return any of those..

Reply:
I think this should be create_dataset.
As it reads in the raw images and it will transform them into new volumes (a lot of .npy files) and store them back on disk.
So that means it creates a new dataset.
Just that this function also creates a datalist that has the entries of the new dataset and returns it.